java/ql/src/semmle/code/java/frameworks/google/Gson.qll

```diff
@@ -0,0 +1,47 @@
+/**
+ * Provides classes for working with the Gson framework.
+ */
+
+import java
+
+/** The class `com.google.gson.Gson`. */
+class Gson extends RefType {
+  Gson() { this.hasQualifiedName("com.google.gson", "Gson") }
+}
+
+/** The class `com.google.gson.GsonBuilder`. */
+class GsonBuilder extends RefType {
+  GsonBuilder() { this.hasQualifiedName("com.google.gson", "GsonBuilder") }
+}
+
+/** A method that registers class types in `GsonBuilder`. */
+class RegsterClassTypeMethod extends Method {
+  RegsterClassTypeMethod() {
```

Comment:
```suggestion
class RegisterClassTypeMethod extends Method {
  RegisterClassTypeMethod() {
```
(Renaming has to be done at usage locations as well)

Reply:
Thanks for pointing out the typo. I've made the change.